src/annotator/adder.tsx

```diff
@@ -200,17 +194,18 @@ export class Adder {
    * - Position the Adder below the selection (arrow pointing up) for LTR selections
    *   and above (arrow down) for RTL selections
    *
-   * @param {DOMRect} selectionRect - The rect of text to target, in viewport
-   *        coordinates.
-   * @param {boolean} isRTLselection - True if the selection was made
-   *        rigth-to-left, such that the focus point is mosty likely at the
-   *        top-left edge of `targetRect`.
-   * @return {Target}
+   * @param selectionRect - The rect of text to target, in viewport coordinates.
+   * @param isRTLselection - True if the selection was made rigth-to-left, such
+   *        that the focus point is mosty likely at the top-left edge of
+   *        `targetRect`.
    */
-  _calculateTarget(selectionRect, isRTLselection) {
+  private _calculateTarget(
```

Comment:
I have updated all methods prefixed with underscore to be strictly private, as I assume that was the intention, and this will ensure the type checker complains if they are accidentally invoked from the outside.

Reply:
Yes, thank you for pushing us on this. We should be doing that.